trulens_eval/trulens_eval/feedback/v2/feedback.py

```diff
@@ -170,14 +172,69 @@ class Groundedness(Semantics, WithPrompt):
     )
 
 
+LIKERT_0_3_PROMPT = "0 to 3, where 0 is the lowest score according to the criteria and 3 is the highest possible score"
+BINARY_0_1_PROMPT = "0 or 1, where 0 is lowest and negative (i.e. irrelevant or not grounded) and 1 is highest and positive (relevant, grounded, valid, etc.)"
+
+
+class OutputSpace(str, Enum):
+    """
+    Enum for valid output spaces of scores.
+    """
+    LIKERT_0_3 = "likert-0-3"
+    BINARY = "binary"
+    # note: we are deprecating the 0 to 10 output space in favor of the likert-0-3 or binary output space
```

Comment:
can we keep 0-10 for a few versions to let this be a slower change for users and so they can compare this side-by-side across different output spaces?

It'd also be good to show our tests on this to see how changing the output space improves calibration

Reply:
yes. I'll work on actual meta-evaluation runs after this PR is done. 

I've added back the likert_0_10 output space as suggested